iso8859_1_to_utf8.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Aura\Filter;
+
+function iso8859_1_to_utf8(string $string): string
```

Comment:
This should be `utf8_to_iso8859_1()`.

Reply:
Done with [1932e6b](https://github.com/auraphp/Aura.Filter/pull/158/commits/1932e6ba4a6ab8c94f19b2fba2d729e7e80456a4) !